classes/person.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :name, :age
+  attr_reader :id, :rentals

```

Comment:
Great job so far. Please change the `rentals` to `attr_accessor` so that it will be both a setter and a getter. you can refer to this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor) for more.